### PR TITLE
update web actions documentation

### PR DESCRIPTION
**Pull request by rabbah**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ install: true
 
 before_install:
   - ./tools/travis/setup.sh
-  
+
```
Comment (csantanapr) [inline .travis.yml]: picky

File: docs/webactions.md [hunk 2]
```diff
@@ -109,26 +109,55 @@ Web actions bring some additional features that include:
 5. `Activation via multiple HTTP verbs`: a web action may be invoked via one of four HTTP methods: `GET`, `POST`, `PUT` or `DELETE`.
 
 
-The example below briefly sketches how you might use these features in a web action. Given an action `/guest/demo/hello` with the following body:
+The example below briefly sketches how you might use these features in a web action. Consider an action `/guest/demo/hello` with the following body:
 ```javascript
 function main(params) { 
-    return { 'response': params};
+    return { response: params };
```
Comment (csantanapr) [inline docs/webactions.md]: picky

File: .travis.yml (hunk 1 above)
Comment (rabbah) [inline .travis.yml]: very

File: docs/webactions.md (hunk 2 above)
Comment (rabbah) [inline docs/webactions.md]: guilty

**Comment by csantanapr** [comment]
merged a93750225b463bf3ac7bf888506421dbda133217

Thank you @cfjedimaster for raising the doc confusion and feedback. Keep them coming

**Pull request closed without merging**